deploy/docker-compose.override.yml

```diff
@@ -1,2 +1,4 @@
 # fork specific overrides
 services:
+  web:
+    pull_policy: always
```

Comment:
If services is empty in override all docker commands will fail with `services must be a mapping`.

Reply:
Wonder what happens if we just have an empty override file...